6-Typescript/server/tests/env/TestKit.ts

```diff
@@ -1,34 +1,41 @@
-import { AppDriver } from '../drivers/AppDriver';
-import axios from 'axios'
-import {TodoDAOMongoDriverTest} from '../drivers/TodoDAOMongoDriverTest'
-import { Application } from 'express';
-import makeApp from '../../app';
-import dotenv from 'dotenv'
+import { AppDriver } from "../drivers/AppDriver";
+import axios from "axios";
+import { TodoDAOMongoDriverTest } from "../drivers/TodoDAOMongoDriverTest";
+import { Application } from "express";
+import makeApp from "../../app";
+import { Server } from "http";
 export class TestKit {
-    appDriver: AppDriver
-    mongoDriver: TodoDAOMongoDriverTest
-    server: any
+    appDriver: AppDriver;
+    mongoDriver: TodoDAOMongoDriverTest;
+    server: Server = new Server();
     constructor() {
-        this.appDriver = new AppDriver(axios, 'http://localhost:5000/api/todos/')
-        this.mongoDriver = new TodoDAOMongoDriverTest()
+        this.appDriver = new AppDriver(
+            axios,
+            "http://localhost:5000/api/todos/"
+        );
+        this.mongoDriver = new TodoDAOMongoDriverTest();
     }
     beforeEach = async () => {
-        const port: number | undefined = Math.ceil((Math.random() * 10000 + 1000))        
-        this.appDriver = new AppDriver(axios, `http://localhost:${port}/api/todos/`)
-        this.mongoDriver = new TodoDAOMongoDriverTest()
-        dotenv.config();
-        const app: Application = makeApp(this.mongoDriver)
-        this.server = await app.listen(port)
-        await this.mongoDriver.connectDB();
+        const port: number | undefined = Math.ceil(
```

Comment:
extract it to another function. it doesn't really have a place in the `beforeEach`
meaning - 
```ts
const port: number|undefined = generatePort();
```

and the logic of this `Math.random() * 10000 + 1000` (which I have to admit that I'm not fully understand) will be in the `generatePort`